graphql/resolvers.go

```diff
@@ -869,6 +869,47 @@ func (r *mutationResolver) SchedulePatchTasks(ctx context.Context, patchID strin
 	return &patchID, nil
 }
 
+func (r *mutationResolver) UnschedulePatchTasks(ctx context.Context, patchID string, abort bool) (*string, error) {
+	modifications := VersionModifications{
+		Action: "set_active",
+		Active: false,
+		Abort:  abort,
+	}
+	err := ModifyVersionHandler(ctx, r.sc, patchID, modifications)
+	if err != nil {
+		return nil, err
+	}
+	return &patchID, nil
+}
+
+func (r *mutationResolver) RestartPatch(ctx context.Context, patchID string, abort bool, taskIds []string) (*string, error) {
+	if len(taskIds) == 0 {
+		return nil, InputValidationError.Send(ctx, fmt.Sprintf("`taskIds` array is empty. You must provide at least one task id"))
+	}
+	modifications := VersionModifications{
+		Action:  "restart",
```

Comment:
can we use the consts declare in graphql/util

Reply:
yup, oversight on my part